diff --git a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
index 3270127..0868085 100644
--- a/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
+++ b/node_modules/react-native-screens/ios/RNSScreenStackHeaderConfig.mm
@@ -38,8 +38,8 @@
 namespace react = facebook::react;
 #endif // RCT_NEW_ARCH_ENABLED
 
-static constexpr auto DEFAULT_TITLE_FONT_SIZE = @17;
-static constexpr auto DEFAULT_TITLE_LARGE_FONT_SIZE = @34;
+static const CGFloat DEFAULT_TITLE_FONT_SIZE = 17.0;
+static const CGFloat DEFAULT_TITLE_LARGE_FONT_SIZE = 34.0;
 
 #if !defined(RCT_NEW_ARCH_ENABLED)
 // Some RN private method hacking below. Couldn't figure out better way to access image data
@@ -488,7 +488,7 @@ + (UINavigationBarAppearance *)buildAppearance:(UIViewController *)vc withConfig
 #endif
 
     NSString *family = config.titleFontFamily ?: nil;
-    NSNumber *size = config.titleFontSize ?: DEFAULT_TITLE_FONT_SIZE;
+    NSNumber *size = config.titleFontSize ?: @(DEFAULT_TITLE_FONT_SIZE);
     NSString *weight = config.titleFontWeight ?: nil;
     if (family || weight) {
       attrs[NSFontAttributeName] = [RCTFont updateFont:nil
@@ -516,7 +516,7 @@ + (UINavigationBarAppearance *)buildAppearance:(UIViewController *)vc withConfig
 #endif
 
     NSString *largeFamily = config.largeTitleFontFamily ?: nil;
-    NSNumber *largeSize = config.largeTitleFontSize ?: DEFAULT_TITLE_LARGE_FONT_SIZE;
+    NSNumber *largeSize = config.largeTitleFontSize ?: @(DEFAULT_TITLE_LARGE_FONT_SIZE);
     NSString *largeWeight = config.largeTitleFontWeight ?: nil;
     if (largeFamily || largeWeight) {
       largeAttrs[NSFontAttributeName] = [RCTFont updateFont:nil
