diff --git a/node_modules/react-native-nitro-modules/android/src/main/cpp/utils/JNISharedPtr.hpp b/node_modules/react-native-nitro-modules/android/src/main/cpp/utils/JNISharedPtr.hpp
index aa71a24..dac6a5d 100644
--- a/node_modules/react-native-nitro-modules/android/src/main/cpp/utils/JNISharedPtr.hpp
+++ b/node_modules/react-native-nitro-modules/android/src/main/cpp/utils/JNISharedPtr.hpp
@@ -62,7 +62,7 @@ public:
                                "> - it's null!");
     }
 #endif
-    return std::shared_ptr<T>(ref->cthis(), GlobalRefDeleter<T>(ref));
+    return std::shared_ptr<T>(ref->cthis(), std::move(GlobalRefDeleter<T>(ref)));
   }
 };
 
diff --git a/node_modules/react-native-nitro-modules/ios/platform/ThreadUtils.cpp b/node_modules/react-native-nitro-modules/ios/platform/ThreadUtils.cpp
index a02b9c8..21d6598 100644
--- a/node_modules/react-native-nitro-modules/ios/platform/ThreadUtils.cpp
+++ b/node_modules/react-native-nitro-modules/ios/platform/ThreadUtils.cpp
@@ -6,7 +6,7 @@
 //
 
 #include "ThreadUtils.hpp"
-#include "UIThreadDispatcher.hpp"
+#include "../threading/UIThreadDispatcher.hpp"
 #include <dispatch/dispatch.h>
 #include <pthread.h>
 #include <sstream>
